Makefile

```diff
@@ -342,6 +342,7 @@ endif
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=pca10040 -opt=1     examples/blinky1
 	@$(MD5SUM) test.hex
+	$(TINYGO) build             -o test.elf  -target=nintendoswitch     examples/nintendoswitch
```

Comment:
Nit: please also add the `@$(MD5SUM) test.hex`.

Reply:
@racerxdl please see this comment.

Reply:
Done